### PR TITLE
Feature/update working farewell wfh

### LINKED ISSUE #424: [ FEATURE REQUEST ] - Update working farewell for WFH
# Overview:
Currently, when saying Goodbye to Natsuki and telling her you are going to work, Natsuki will assume the player is heading out somewhere to work; as seen when she requests the player take care.

Natsuki should ask if the player is working from home or heading out, so she can wish them goodbye appropriately!

## Must haves:
- Natsuki must distinguish between the player leaving for a workplace, and working from home when wishing the player goodbye: her dialogue must vary accordingly

## Could haves:
N/A

## Is this feature request related to a problem? Please describe.
This is a new feature.

### PR DESCRIPTION
**Pull request by Blizzardsev**
Introduces the following:
- Update to `farewell_option_work` to account for a player WFH when it is late.
- Update to `farewell_option_work` to account for player birthday.
- Most routes unchanged, as they are not assuming the player is going somewhere: just that they will be back later.

Closes #424

### CODE REVIEW

**Review by Nemu-sus — APPROVED** [review]

File: game/script-farewells.rpy [hunk 1]
```diff
@@ -345,17 +345,45 @@ label farewell_option_work:
...
+        n 1kllajl "Are you..."
+        show natsuki 1tnmbol
+
+        menu:
+            n "Are you at {i}least{/i} working from home today,{w=0.2} or...?"
+
+            "Yes, I'm working from home.":
+                n 1ncsssl "Heh.{w=0.75}{nw}"
+                extend 1nllpul " I {i}guess{/i} that's something,{w=0.75}{nw}"
+                extend 1nslsll " at least."
```
Comment (RaionArt) [inline game/script-farewells.rpy:357]: I'm wondering if it comes off a bit uninspired with relying on the word "at least" twice or if you just wanted to highlight this expression

**Review by RaionArt — COMMENTED** [review]
Good job on including even the birthday!

File: game/script-farewells.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/script-farewells.rpy:357]: Ah, good catch - I need to correct that

**Review by Nemu-sus — APPROVED** [review]
👍

**Review by RaionArt — APPROVED** [review]

**Pull request merged**